tensorflow/java/src/main/java/org/tensorflow/op/Scope.java

```diff
@@ -154,6 +154,19 @@ public Scope withName(String opName) {
   public String makeOpName(String defaultName) {
     return nameScope.makeOpName(defaultName);
   }
+  
+  /**
+   * The name prefix of this scope.
+   * <p>
+   * This value is the combination of the name of this scope and all of its parents, seperated by a '/', e.g.
```

Comment:
Should we add a test for `prefix`, particularly one that exercises the effects of `NameScope.makeUnique`?